integrasjon/pom.xml

```diff
@@ -26,6 +27,11 @@
         <module>spokelse-klient</module>
         <module>kafka-properties</module>
     </modules>
+
+    <properties>
+        <graphql-java-codegen.version>5.7.1</graphql-java-codegen.version>
```

Comment:
Bør du ha denne i rot-pom? Lurer på om det også ligger en dep til versjonsnr i fp-bom .....
Vet ikke konvensjon men har sett at dependabot ikke alltid scanner alle under-poms

Reply:
Jeg skal slette den fra fp-bom siden det er kun felles og vtp som bruker den.

Og det er greit at den ligger i felles integrasjon siden disse deps er kun gjeldende for klienter.

Reply:
Generelt maven regel er at ting definert direkte i prosjektet overskriver alt `arv` så fp-bom vil ikke ha mye å si her.

Reply:
Lurte bare fordi dependabot ikke oppdaget gammel versjon av jose4j som lå i felles/felles eller felles/felles/oidc - flyttet den høyere opp.

Reply:
Det jeg har sett et par ganger at om man lukker en PR som dependabot oppretter manuelt så ignorerer dependabot alle `patch` updates på dette biblioteket til en ny `minor` versjon kommer.

Reply:
Denne ligger heler ikke i rot-pomen her https://github.com/navikt/fp-felles/pull/1271 så jeg regner med at det vil fungere greit.